website/src/views/pageStateHandlers/WasapPageStateHandler.ts

```diff
@@ -192,7 +200,10 @@ export class WasapPageStateHandler implements PageStateHandler<WasapFilter> {
                 break;
             case 'untracked':
                 setSearchFromString(search, 'sequenceType', analysis.sequenceType);
-                setSearchFromString(search, 'excludeVariants', analysis.excludeVariants?.join('|'));
+                setSearchFromString(search, 'excludeSet', analysis.excludeSet);
+                if (analysis.excludeSet === 'custom') {
+                    setSearchFromString(search, 'excludeVariants', analysis.excludeVariants?.join('|'));
```

Comment:
Do we need a check that the URL doesn't get too long? (Same issue that Loculus has)

Reply:
Created #848 for this; I think it's a good, valid point and we need to tackle it, but it's not specific to the changes in this PR (the free-text field already existed before).